gemsfx/src/main/java/com/dlsc/gemsfx/skins/TimePickerSkin.java

```diff
@@ -94,19 +104,6 @@ public TimePickerSkin(TimePicker picker) {
                     updateSecondsMillisecondsViewable(false, false);
                     break;
             }
-        }));
-
-        updateEmptyPseudoClass();
```

Comment:
why was this all removed?

Reply:
I put it back. Initially all I thought it was doing was the sys outs. I removed the sys out statements.